server/routes/api/documents/documents.ts

```diff
@@ -923,7 +932,7 @@ router.post(
   validate(T.DocumentsTemplatizeSchema),
   transaction(),
   async (ctx: APIContext<T.DocumentsTemplatizeReq>) => {
-    const { id } = ctx.input.body;
+    const { id, publish, workspace } = ctx.input.body;
```

Comment:
In terms of an interface I think it would be better to allow `collectionId` in this payload – you could pass any `collectionId` to templatize into that specific collection (irregardless of the current collection), or pass `collectionId=null` to represent no collection (workspace)

This would also be better aligned with how draft documents work

Reply:
Along with this, the templatize dialog can have a "collection" selection instead of defaulting to the current one. Your thoughts?

Reply:
Yes, I think so – It can default to the current collection and allow selection of another or "Workspace"